snorkel/preprocess/core.py

```diff
@@ -6,7 +6,7 @@
 pipeline. This allows LFs to share common preprocessing steps that
 make it easier to express labeling logic. A simple example for text
 processing is concatenating the title and body of an article. For a
-more complex example, see ``snorkel.labeling.preprocess.nlp.SpacyPreprocessor``.
+more complex example, see ``snorkel.preprocess.nlp.SpacyPreprocessor``.
```

Comment:
this docstring contains specific text about LFs. change to "Snorkel operators"

Reply:
and do a general pass for this

Reply:
thanks for the catch!